### DISCUSSION TITLE
Mouse Control

### DISCUSSION THREAD

**Discussion by animafps** [aiming-taxonomy/glossary#2] [Concepts]
# Mouse Control

Current definition: 
`The ability to move the mouse and respectively the crosshair to the position you want smoothly or to stay on target and with speed and many other factors that make good aim and to adapt mouse movements based on visual or audio feedback.`

**Comment by riddbtw** [comment]
I think I've been using the term mouse control incorrectly over most others.  I think of mouse control purely as your physical ability using the muscles in your shoulder, arm, hand, wrist, and fingers to correctly position your crosshair on screen.  So if you think of the crosshair as a physical extension of your body, this is your ability to manipulate it.  I then separated out sub-components like reactivity, smoothness, microadjusting which feed into mouse control to allow conversations on improving specific parts of people's aim.  Ended up calling everything combined 'raw aim' realizing this isn't a common definition.

The only reason why I separated it out is because I saw concepts like reading and hand-eye coordination which doesn't have to do with physical manipulation.  But they may be the only ones and could just be separated on their own.  Either way I'd be fine with just stating "here's the things you can train to build mouse control, here's these other related items' and just abandon the idea of 'raw aim' altogether.  

Do we consider mouse control = mousefeel? I think yes.  

General note: I think we should avoid in any definition using terms like: etc, 'many others,' 'some examples'
Just because if we're building a taxonomy, unless there's infinite permutations we should strive to define everything, and just list them all out.

**Comment by deLiGHT-03** [comment]
I consider mouse control as the mechanical ability to accurately move the mouse with intention.

If you can't move the mouse at the speed you want with the precision you need, you're lacking in mouse control.

**Reply by animafps** [reply]
speed != precision != accuracy. is the intention the same as the requirements of the situation

**Reply by deLiGHT-03** [reply]
Not sure I follow. Particularly the second part. Can you expand?

**Reply by animafps** [reply]
you defined it with accurately then used speed and precision in your example. and you said with intention but again in your example you used "the precision that you need" as well as me just asking because that part of your definition can be better phrased and less vague

**Reply by deLiGHT-03** [reply]
Intention was meant to encompass variables of speed and precision. I was just explaining it a little more in the follow-up as such. 

Accurately was in reference to the execution of the intention but I can see how that can confuse terms. Better to say "successfully" perhaps. If you are successfully moving your mouse at the speed and precision you need (to fulfill your intention), you are meeting mouse control demands on a task. 

I say intention because it's all relative. Not every task has a set mouse control demand, but every task is open to your mouse intention and your ability to achieve that intention defines mouse control success/failure.
